modules/swoop/swoop-bundle/main.tf

```diff
@@ -1,69 +1,74 @@
-resource "helm_release" "swoop_api" {
-  name = "swoop-api"
```

Comment:
Is it not possible now to deploy just the Swoop API helm chart if someone wants to do that? Do they only have the option of deploying the only the Swoop bundle, only postgres, or only minio, but not only swoop-api?

Reply:
Is this because the Swoop API needs postgres and/or minio in its initialization, and would fail to do so if we also didn't deploy postgres and/or minio in the form of a swoop-bundle, and the above scenario would never arise?

Reply:
I see that there is a separate helm chart for swoop-api in the e84 helm repo, and if that is not going to be supported anymore via Terraform, we can remove it?

Reply:
It is possible now to deploy just the swoop-api within the swoop-bundle helm chart, that is what this block is for: https://github.com/Element84/filmdrop-k8s-tf-modules/blob/hm/swoop-bundle/modules/swoop/swoop-bundle/main.tf#L10-L13

Reply:
No, the helm repo still uses the swoop-api within the swoop-bundle as a dependency: https://github.com/Element84/filmdrop-k8s-helm-charts/blob/main/charts/swoop-bundle/Chart.yaml#L11-L14

Reply:
But isn't that still deploying the `swoop-bundle `helm chart and not the `swoop-api `helm chart? Is the `swoop-api `helm chart being used anymore?

Reply:
It is deploying the swoop-api helm chart, due to the swoop-bundle installing the swoop-api as a dependency because of the `swoop-api.enabled = true` condition. The idea to maintain the individual helm charts, would be that each individual helm chart can have their own release cadence, versioning and their own dependencies. The swoop-bundle would be a grouping of swoop-components that are compatible with the same swoop-db schema version.